### ISSUE TITLE
[Feature Request] Option for changing fonts

### ISSUE THREAD

**Issue by samuelsarao** [federicoiosue/Omni-Notes#803]
Can we have the option to change the current serif font to a sans serif font? It's more easy on eyes and easy to read than serifs :)

**Comment by federicoiosue** [comment]
Good point. I really believe I'll move to more modern sans serif font when the app's redesign will come.

At the time I choose serif fonts due to non-technical nature of the app but now I'd love a more polished design myself!

**Comment by samuelsarao** [comment]
> Good point. I really believe I'll move to more modern sans serif font when the app's redesign will come.
> 
> At the time I choose serif fonts due to non-technical nature of the app but now I'd love a more polished design myself!

This is exciting! Thank you so much!

**Comment by trustable-code** [comment]
+1. With a sans serif font it would be the best notes app for me.

**Comment by andr222** [comment]
I like  Deja Vu sans mono, which has a lower case L that looks like an L.  (The same as vera sans mono).  Trébuchet is a proportional version of that font, for those who don't like monospace fonts.
